pkg/client/record/event.go

```diff
@@ -25,11 +26,23 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/wait"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/watch"
 
 	"github.com/golang/glog"
 )
 
+const (
+	maxQueuedEvents  = 1000
+	maxTriesPerEvent = 10
+)
+
+var (
+	minSleep   = float64(1 * time.Second)
+	maxSleep   = float64(15 * time.Second)
+	backoffExp = 1.5
```

Comment:
Is it really an issue that it isn't easy to compute how long it'll be before an event gets dropped? Exponential backoff is common because it provides good real world behavior, not because it's easy to compute that total wait time.

For what it's worth, the total here is on the order of 2 minutes. but wait.Jitter adds a lot of variability anyways, since it adds an additional random fraction of the input wait time onto its result.